### ISSUE TITLE
My test script is as follows

### ISSUE THREAD

**Issue by SONGjiemo** [quoid/userscripts#232]
My test script is as follows:
1. Unable to click [Custom] to set
![image](https://user-images.githubusercontent.com/46627639/156936808-9d6d18d2-3e04-42f6-9853-3fdcf0d95602.png)

https://greasyfork.org/zh-CN/scripts/14178-ac-baidu-%E9%87%8D%E5%AE%9A%E5%90%91%E4%BC%98%E5%8C%96 %E7%99%BE%E5%BA%A6%E6%90%9C%E7%8B%97%E8%B0%B7%E6%AD%8C%E5%BF%85%E5%BA%94%E6 %90%9C%E7%B4%A2-favicon-%E5%8F%8C%E5%88%97
2. The script cannot be loaded
![image](https://user-images.githubusercontent.com/46627639/156936929-49046776-e756-4606-97f5-a12f6f8aa078.png)

https://greasyfork.org/en-US/scripts/27752-searchenginejump-%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E%E5%BF%AB%E6 %8D%B7%E8%B7%B3%E8%BD%AC
3. The loading does not seem to be successful, and the function is not successful
https://greasyfork.org/en-US/scripts/422814-%E7%99%BE%E5%BA%A6%E4%BA%91%E5%8E%BB%E5%B9%BF%E5%91 %8A-%E5%BF%AB%E6%8D%B7%E9%94%AE%E5%80%8D%E9%80%9F-%E5%85%8Dvip%E8%A7%A3%E9%94 %81%E5%80%8D%E9%80%9F-%E7%94%BB%E8%B4%A8-%E5%85%A8%E7%BD%91%E7%8B%AC%E5%AE %B6

4.The script cannot be loaded
https://greasyfork.org/zh-CN/scripts/12909-google-baidu-switcher-all-in-one

_System Information:_
> macOS or iOS version: IOS
> Userscripts version: NO
> Safari version: NO
> Is this issue related to script injection?    I have no idea
> Did the test script (pasted above) successfully run on your machine?   Oil monkey can be successful, but the tool on IOS does not work

**Comment by quoid** [comment]
I looked at the [userscript](https://greasyfork.org/scripts/14178-ac-baidu-重定向优化百度搜狗谷歌必应搜索-favicon-双列/code/AC-baidu-重定向优化百度搜狗谷歌必应搜索_favicon_双列.user.js) and tried to install it. I notice there are certain api methods that are not **yet** supported:

```
@grant
GM_getValue
GM.getValue
GM_setValue
GM.setValue
GM_addStyle
GM_getResourceURL **not supported**
GM_listValues
GM_getResourceUrl **not supported**
GM.getResourceUrl **not supported**
GM_xmlhttpRequest
GM_getResourceText **not supported**
GM_registerMenuCommand **not supported**
unsafeWindow
```

Related tasks: https://github.com/quoid/userscripts/issues/224 https://github.com/quoid/userscripts/issues/230

When I run the script on https://www.baidu.com, I notice errors due to the unsupported apis:
<img width="1455" alt="Screen Shot 2022-03-06 at 1 42 32 PM" src="https://user-images.githubusercontent.com/7660254/156937413-55ce7756-c47e-468b-9585-3ff0091dd170.png">

**Until the missing API methods are updated, this userscript will not function, unless the author updates the userscript to allow usage without those methods.**

我正在使用翻译器，为糟糕的语法道歉。 问题是目前不支持使用的 api 方法。 我计划在未来的更新中支持这些方法。 请让我知道这是否被理解。

**Comment by SONGjiemo** [comment]
I got it, thank you very much.

**Comment by quoid** [comment]
closing this issue as the unsupported API methods will be fixed in #224 

我正在关闭这个问题，因为问题已经存在于另一个问题中 - 这是一个自动翻译
